src/utils/bibUtils.ts

```diff
@@ -104,12 +99,13 @@ export function getBibQueryString(
         .join("")
     : ""
 
-  const paginationQuery = `items_size=${batchSize}&items_from=${itemsFrom}&item_page=${itemPage}`
+  const paginationQuery = bibQuery?.all_items
```

Comment:
I think a better name would be `paginationOrAllQuery`

Reply:
Sounds good i like that naming better